src/FileLogger.jl

```diff
@@ -14,7 +14,8 @@ struct FileLogger <: _iologger
     flush::Bool
     append::Bool
 
-    FileLogger(logPaths::Dict{LogLevel, String} = Dict(Info => "default.log"); flush = true, append = true) = new(logPaths, Dict{LogLevel,IO}(), Dict{Any, Int}(), flush, append)
+    FileLogger(logPaths::Dict{LogLevel, String} = Dict(Info => "default.log"); flush = true, append = true) =
```

Comment:
I think the newline here is from your editor settings?

Reply:
I have cancelled my changes to FileLogger.jl